fusionfrontend/src/assets/sass/components/_primetable.scss

```diff
@@ -165,4 +165,78 @@
   }
 }
 
+.header-line {
+  height: 50px;
+  margin: 30px 80px;
+
+  .table-header{
+    font-size: $big-font-size;
+    font-weight: bold;
+    margin-top: 14px;
+    margin-left: 68px;
+  }
+
+  .filter-btn-div {
+    position: relative;
+    margin-top: .5rem;
+    height: 50px;
+    padding-left: 10px;
+  }
+
+  .create-btn {
+    margin-top: .5rem;
+    align-self: center
+  }
+
+  .create-btn-without-filter {
+    align-self: center;
+    margin-right: 48px;
+  }
+}
+
+
+.table-dropdown-menu-button {
+  outline: none;
+  background-color: transparent !important;
+  color: #95989a !important;
+  border: none !important;
+}
+
+.table-dropdown-menu {
+   .p-menu {
+    width: 220px;
+    border-radius: 10px !important;
+    padding-top: 0.5em !important;
+    padding-bottom: 0.5em !important;
+    margin-top: 0.25em !important;
+  }
+
+   .p-menu-overlay {
+    top: unset !important;
+  }
+
+   .p-menuitem-link {
+    height: 40px !important;
+    font-size: 14px !important;
```

Comment:
use variable for font-size and extract values of font-family to new variable and add it on all places